docs/en/modules/node.md

```diff
@@ -131,6 +127,38 @@ node.dsleep(nil,4)
 - [`wifi.suspend()`](wifi.md#wifisuspend)
 - [`wifi.resume()`](wifi.md#wifiresume)
 - [`node.sleep()`](#nodesleep)
+- [`node.dsleepMax()`](#nodedsleepmax)
+
+## node.dsleepMax()
+
+Returns the current theoretical maximum deep sleep duration.
+
+!!! caution
+
+	While it is possible to sleep longer than the theoretical maximum sleep duration, which can be found with `node.dsleepMax()`, it is not recommended to exceed this theoretical maximum. 
```

Comment:
Did you test this? My [own tests](https://thingpulse.com/max-deep-sleep-for-esp8266/) conducted with Arduino core showed that the chip would sleep indefinitely if you went beyond. Even if you use dsleep max it will wake up before time is up (again in my tests).

Reply:
I did... but I only got it to wake up once and it registered as an external reset(6) rather than a wake from deep sleep() and I made a mistake in the logging code, so times weren't recorded. I will have to do more testing.